dot/sync/chain_sync_test.go

```diff
@@ -1246,7 +1247,11 @@ func Test_chainSync_start(t *testing.T) {
 	}
 }
 
+var blockAnnounceHeader = &types.Header{Number: 2}
+
```

Comment:
nit Move this locally in the test(s) where it's needed, remove from global scope :pray: